venia-integration-tests/package.json

```diff
@@ -5,12 +5,13 @@
     "scripts": {
         "clearCache:mac": "rm -rf ~/Library/Application\\ Support/Cypress/cy/*",
         "test": "cypress open --browser chrome --config-file cypress.config.json",
+        "test:ci": "sh run-tests.sh",
         "test:debug": "node --inspect-brk ./node_modules/cypress/bin/cypress run --browser chrome --config-file cypress.config.json",
         "test:devTest": "jest",
         "test:firefox": "cypress open --browser firefox --config-file cypress.config.json",
-        "test:headless": "cypress run --browser chrome --headless --config-file cypress.config.json",
+        "test:headless": "sh run-tests.sh",
```

Comment:
rather than run these commands with `sh`, you specify `bash` in the scripts itself `#!/usr/bin/env bash`, so run `chmod a+x` on both files and then you can change these commands to `./run-tests` and drop the `.sh` from the end.